dbt/macros/aggregate_cdu.sql

```diff
@@ -3,7 +3,9 @@
     select
         parid,
         taxyr,
-        array_join(array_sort(array_distinct(array_agg({{ cdu_column }}))), ', ') as cdu
+        array_join(
+            array_sort(array_distinct(array_agg(upper({{ cdu_column }})))), ', '
```

Comment:
Apparently there are some lowercase CDUs, as well. Might as well fix that while we're in here. 🤦